web/src/css/6-components/_know-your-rights-route.scss

```diff
@@ -0,0 +1,21 @@
+.RightsList {
+    font-family: Times;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    padding: 0.6rem;
+}
+
+.RightsItem h6 {
+    font-weight: bold;
+    font-size: 1.2em;
+    padding: 0.6rem;
+    display: block;
+
+}
+
+.RightsItem li {
+    padding-left: 1rem;
+    padding-right: 1rem;
+    margin: 0.5rem;
+}
```

Comment:
Nit: Add a newline at the end of the file. It's kinda arcane, but it's because various unix commands expect all files to end with a newline so they might not work correctly if the file doesn't end in a newline.